x/farming/client/rest/rest.go

```diff
@@ -0,0 +1,24 @@
+package rest
+
+import (
+	"net/http"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	govrest "github.com/cosmos/cosmos-sdk/x/gov/client/rest"
+)
+
+// TODO: not implemented yet; although this legacy REST API may not be needed,
+// it needs for ProposalHandler; otherwise it throws panic when starting the chain
+
+// ProposalRESTHandler returns a ProposalRESTHandler that exposes the community pool spend REST handler with a given sub-route.
```

Comment:
Need to update the comments

Reply:
Yes, I will update the comment as the following comment.
> // ProposalRESTHandler returns a ProposalRESTHandler that exposes the farming plan proposal (add/update/delete) REST handler with a given sub-route.

FYI, I was thinking of removing rest package since this code is legacy but then I bring it back to prevent from getting `panic` when starting a new chain.  Do you think it is necessary to have this REST API developed?